app/views/catalog/index.html.erb

```diff
@@ -5,18 +5,21 @@
     <%= render partial: 'no_equipment' %>
   <% else %>
     <div class="row">
-      <div id="pagination_line" class="col-md-12">
+      <div id="pagination_line" class="col-md-10">
         <%# for kaminari pagination %>
         <%= paginate @page_eq_models_by_category,
               params: { controller: 'catalog', action: 'index' } %>
-        <%= form_tag update_user_per_cat_page_path,
+	<%= form_tag update_user_per_cat_page_path,
```

Comment:
I think it just converted the spaces to tabs to match the rest of the file. To be honest, I think it's fine.